doc/cs_registers.rst

```diff
@@ -510,6 +522,20 @@ Other bit fields read as zero.
 |       |      | parameter == 0), this field will always read as zero.            |
 +-------+------+------------------------------------------------------------------+
 
+Security Feature Seed Register (secureseed)
+-------------------------------------------
+
+CSR Address: ``0x7C1``
+
+Reset Value: ``0x0000_0000``
+
+Accessible in Machine Mode only.
+
+Custom CSR to allow re-seeding of security-related pseudo-random number generators.
+A write to this register will update the seeding of pseudo-random number generators inside the design.
+This allows software to improve the randomness, and therefore security, of certain features by periodically reading from a true random number generator peripheral.
+Seed values are not actually stored in a register and so reads to this register will always return zero.
```

Comment:
Agreed there isn't any point in reading it, but I guess it's also not an illegal thing (that should cause a piece of software to trap and possibly be abandoned) either. As you said, I mostly did it this way because there isn't a concept of write-only CSRs in the spec, but maybe we should think about this further.